spec/models/user_spec.rb

```diff
@@ -1,5 +1,74 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user) { build(:user) }
+  let!(:next_user) { create(:user) }
+  let(:attributes_presence) { %i[first_name last_name phone project_id email] }
+
+  context 'when got project id' do
+    it 'should get error message if id absent' do
+      id = 22_423
+      expect(Project.exists?(id)).to eql(false)
+    end
+
+    it 'should have associated projects via id' do
+      id = 2
```

Comment:
Also here.. You shouldn't have use id.
Use your factories to operate with data in test.